front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -170,28 +170,22 @@ const ImportTrainScheduleConfig = ({
     startDate: string
   ): Step[] =>
     ocpTTs
-      .map((ocpTT, index): Step | null => {
+      .map((ocpTT): Step | null => {
         const ocpRef = ocpTT.getAttribute('ocpRef');
         const times = ocpTT.getElementsByTagName('times')[0];
         let departureTime = times?.getAttribute('departure') || '';
         let arrivalTime = times?.getAttribute('arrival') || '';
 
-        const isLastOcpTT = index === ocpTTs.length - 1;
-
-        if (isLastOcpTT) {
-          arrivalTime = cleanTimeFormat(departureTime) || cleanTimeFormat(arrivalTime); // For the last sequence, arrival equals departure
-          departureTime = cleanTimeFormat(arrivalTime) || cleanTimeFormat(departureTime);
-        } else if (index !== 0) {
-          arrivalTime = times?.getAttribute('arrival') || times?.getAttribute('departure') || '';
-          arrivalTime = cleanTimeFormat(arrivalTime);
-          departureTime = times?.getAttribute('departure') || times?.getAttribute('arrival') || '';
-          departureTime = cleanTimeFormat(departureTime);
-        }
+        arrivalTime = times?.getAttribute('arrival') || times?.getAttribute('departure') || '';
```

Comment:
Seems like these variables are set, and then always overwritten with something else? Probably we can drop the first value they're set to?